source/fab/language/fortran.py

```diff
@@ -31,10 +30,61 @@
 from fab.reader import TextReader, TextReaderDecorator
 
 
+class FortranUnitUnresolvedID(object):
+    def __init__(self, name: str):
+        self.name = name
+
+    def __eq__(self, other):
+        if not isinstance(other, FortranUnitUnresolvedID):
+            message = "Cannot compare FortranPartialUnitID with " \
```

Comment:
Should this message say `FortranUnitUnresolvedID`?  (feels like an earlier iteration of the name)

Reply:
Of course you are right. What you are seeing there is a left over from a previous name for the class.